### PR TITLE
Feature setup calculator

### PR DESCRIPTION
**Pull request by AlejandroNo4**
At this milestone, I created the project structure setup. I added Big.js and prop-types using npm.
I deleted the unnecessary files.
I also deployed the app on [Heroku](https://calculator-no4.herokuapp.com). The Heroku page is empty, as only the structure was created.

Thank you in advance.

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Invalid :red_circle: 

- Please make sure that Github action is running
- Your readme is descripitive. You need simply to include the title of the project, a description about it, and instructions on how to setup and run the project. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Invalid red_circle

Please make sure that Github action is running. You can use this [repo](https://github.com/microverseinc/linters-config/tree/master/react-redux)

File: README.md [hunk 1]
```diff
@@ -1,76 +1,75 @@
...
 
-### Install
+- Now that you have the repository in your local machine. Change the direction of the local file with your terminal. Just type "$ cd" (without the $ sign) and the path to reach the calculator-app folder.<br/>
+  Something like:
 
-### Usage
+        $ cd User/Documents/calculator/calculator-app
 
-### Run tests
+- You are done with the installations! Now to run the project, first type:
```
Comment (rindrajosia) [inline README.md:43]: - [x] [REQUIRED] It will be nice if you ask users to install packages by using `npm install` before` npm start`

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- Good job by removing unnecessary files :+1:
- app runs correctly with `npm start` :+1: 

To improve :factory_worker: [REQUIRED]:

- [x] [REQUIRED] Please add `big.js` and `prop-types` to package.json. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/big.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/section/propTypes.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

- [x] [REQUIRED] Your live demo link is broken. It will be nice if you fix it.

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by bellom — APPROVED** [review]
## Hi @AlejandroNo4  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Pull request merged**